CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,7 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [vmdk] Fix `VBOX_E_FILE_ERROR (0x80BB0004)` when trying to copy a VMDK exported from XO in virtual box (PR [#6163](https://github.com/vatesfr/xen-orchestra/pull/6163))
```

Comment:
1. You missed an occurrence of `vmdk`
2. `virtual box` → `VirtualBox`
3. *import* a VMDK in VirtualBox?
4. 
```suggestion
- [VMDK Export] Fix `VBOX_E_FILE_ERROR (0x80BB0004)` when importing in VirtualBox (PR [#6163](https://github.com/vatesfr/xen-orchestra/pull/6163))
```